Source/WebCore/Modules/geolocation/Geolocation.cpp

```diff
@@ -158,7 +158,7 @@ SecurityOrigin* Geolocation::securityOrigin() const
 
 Page* Geolocation::page() const
 {
-    return document() ? document()->page() : nullptr;
+    return document() ? RefPtr { document() }->page() : nullptr;
```

Comment:
Instead of calling document twice. Can we do this instead?
```
RefPtr document = this->document();
return document ? document->page() : nullptr;
```